### PR TITLE
Fix for albums without publication year

### PR DESCRIPTION
**Pull request by grray**
This PR fixes get_artist_albums if some albums don't have publication year. For example for https://music.youtube.com/channel/UCfUu_pLg6Uw-x_ntBWZmlwA

### CODE REVIEW

**Comment by sigma67** [comment]
I don't see any albums without publication year on that page, are you sure you linked the right artist? If yes, can you provide a screenshot?

**Comment by grray** [comment]
It's at the bottom of album list
image

**Comment by grray** [comment]
By the way, how is better for you, put these small "missing data" fixes each in separate PR, or make one big?

File: ytmusicapi/parsers.py [hunk 1]
```diff
@@ -307,9 +307,13 @@ def parse_song_album(data, index):
     }
 
 
-def get_item_text(item, index, run_index=0):
+def get_item_text(item, index, run_index=0, none_if_absent=False):
     column = get_flex_column_item(item, index)
-    return column if not column else column['text']['runs'][run_index]['text']
+    if not column: 
+        return None    
+    if none_if_absent and len(column['text']['runs']) < run_index+1:
```
Comment (sigma67) [inline ytmusicapi/parsers.py:314]: Can you explain the need for ` and len(column['text']['runs']) < run_index+1` ?

Comment (grray) [reply ytmusicapi/parsers.py:314]: It's checking, how many items array has, to make sure that column['text']['runs'][run_index] exists. Perhaps there is better way to do this, python is not my main programming language :)

**Pull request merged**